cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConductingEquipmentConversion.java

```diff
@@ -554,6 +554,16 @@ public void connect(BranchAdder<?> adder,
         connect(context, adder, iidmVoltageLevelId1, busId1, t1Connected, node1, iidmVoltageLevelId2, busId2, t2Connected, node2);
     }
 
+    public static void connect(Context context, InjectionAdder<?> adder, String busId, boolean connected, int node) {
+        if (context.nodeBreaker()) {
+            adder.setNode(node);
+        } else {
+            adder
+                    .setBus(connected ? busId : null)
+                    .setConnectableBus(busId);
+        }
+    }
+
     public static void connect(Context context, BranchAdder<?> adder,
         String iidmVoltageLevelId1, String busId1, boolean t1Connected, int node1,
         String iidmVoltageLevelId2, String busId2, boolean t2Connected, int node2) {
```

Comment:
This method is also not used anymore in the PowSyBl project. Maybe we can deprecate it.

Reply:
Same comment

Reply:
Done